src/system/ExtensionsModule/Resources/views/ExtensionsInterface/footer.html.twig

```diff
@@ -0,0 +1 @@
+{# Not used #}
```

Comment:
Yes, hard to say one side of me is saying that it might be useless and the other is looking for ways of using it:) well this is symmetric function to adminfooter nobody is using it, I think not even core, but let say that logical consequence of having header is to have footer as well - at least in my world ;)